cli/lib/gateway.js

```diff
@@ -24,7 +24,7 @@ Gateway.prototype.start = function start(options, cb) {
   const args = { target: cache, keys: keys,pluginDir:options.pluginDir };
   const that = this;
   if (cluster.isMaster) {
-    edgeconfig.get({ source: source, keys: keys }, function (err, config) {
+    edgeconfig.get({ source: source, keys: keys, localproxy: {name:options.app_name , path:options.app_path, target_url:options.target_url} }, function (err, config) {
```

Comment:
Won't this cause your conditional here to always be true, always preferring localproxy?

https://github.com/apigee/microgateway-config/pull/6/commits/11f77a619a840cdca59bb3dc7ecea037435d136a#diff-9a3ce09856114605620e98d815bd93e1R56